docs/tools_user_guide.md

```diff
@@ -322,6 +335,11 @@ You can also leverage model hashes (see [Model Hashes](#model-hashes)) to filter
 
 > See the [Filtering Model Hashes tutorial](https://github.com/groq/mlagility/blob/main/examples/cli/discovery.md#filtering-model-hashes) for a detailed example.
 
+Additionally, you can leverage labels (see [Labels](#labels)) to filter which models in a script will be acted on, in the following manner:
+  - `benchit *.py --label test_group::a` will only benchmark the scripts labels with `test_group::a`.
```

Comment:
Perhaps consider calling it `--labels` instead since it supports multiple labels?

Reply:
Switched to `--labels`